sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_client.py

```diff
@@ -294,7 +294,7 @@ def restore_secret_backup(self, backup, **kwargs):
 
     @distributed_trace
     def begin_delete_secret(self, name, **kwargs):
-        # type: (str, **Any) -> DeletedSecret
+        # type: (str, **Any) -> KeyVaultOperationPoller
```

Comment:
KeyVaultOperationPoller is a private type since it's in `_shared`, so we should list the return type as an LROPoller:
```suggestion
        # type: (str, **Any) -> LROPoller
```